static/openapi/eds-spec.json

```diff
@@ -0,0 +1,399 @@
+              "X-PhotonIQ-CustomerId": {
+                "description": "Customer ID header.",
+                "schema": {
+                  "type": "string",
+                  "description": "Customer ID that includes the tenant and fabric name."
+                }
+              }
+            }
+          },
+          "400": {
+            "description": "Bad request.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "object",
+                  "properties": {
+                    "error": {
+                      "type": "string",
+                      "description": "Error message."
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/es/v1/metrics": {
+      "get": {
+        "summary": "Retrieve Metrics",
+        "description": "Retrieve metrics data within a specified time range.",
+        "tags": [
+          "Event Delivery Metrics"
+        ],
+        "parameters": [
+          {
+            "in": "query",
+            "name": "start",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The start date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "end",
+            "required": true,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The end date used for querying metrics data. Must be provided in RFC3339 date-time format.",
+            "example": "YYYY-MM-DDTHH:MM:SSZ"
+          },
+          {
+            "in": "query",
+            "name": "name",
+            "required": false,
+            "schema": {
+              "type": "string"
+            },
+            "description": "The name of the metrics data to retrieve."
+          },
+          {
+            "in": "query",
+            "name": "limit",
+            "required": false,
+            "schema": {
+              "type": "integer"
+            },
+            "description": "The maximum number of metrics to retrieve (optional)."
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved metrics.",
+            "content": {
+              "application/json": {
+                "schema": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "domain_key": {
+                        "type": "string",
+                        "description": "The domain key."
+                      },
+                      "es_cpu_usage_second": {
+                        "type": "number",
+                        "description": "ES CPU usage per second."
```

Comment:
What is ES CPU usage?  Is that different from CPU usage?

What does this number mean?  If it says 5, what does 5 mean?

Reply:
Its the CPU usage. Have removed the ES text as it was not required. I believe this number indicates percentage like it's shown in telemetry. 5s would mean 500% CPU usage.